fw/http_msg.c

```diff
@@ -566,7 +566,7 @@ tfw_http_msg_hdr_close(TfwHttpMsg *hm)
 	parser->hdr.flags |= TFW_STR_COMPLETE;
 
 	/* Cumulate the trailer headers length */
-	if (parser->hdr.flags & TFW_STR_TRAILER) {
+	if (is_srv_conn && parser->hdr.flags & TFW_STR_TRAILER) {
```

Comment:
Maybe to aggregate this `if` with the one at the below?
```C
if (is_srv_conn) {
    if (parser->hdr.flags & TFW_STR_TRAILER) {
       ....
     if (!TFW_MSG_H2(hm) && ....
 ```

Reply:
The if below is for `!is_srv_conn`